.github/workflows/testConfig.yml

```diff
@@ -0,0 +1,47 @@
+name: Test Config
+
+on:
+  pull_request:
+    paths:
+      - 'src/__tests__/config/**'
```

Comment:
Here is the line that runs this workflow only when a file changes in the config folder.